### ISSUE TITLE
Allow passing R arguments to `rig run`

### ISSUE THREAD

**Issue by AdrienLeGuillou** [r-lib/rig#249]
Hi, 

is there a way to pass `R` arguments like `--no-save` to `rig run`?

something like `rig run -- --no-save`

Thanks

**Comment by AdrienLeGuillou** [comment]
I do get the `Save workspace image? [y/n/c]: ` prompt when quitting with Ctrl-D (or the signal sent by iron.nvim) and as well with just `q()`.

I can make an alias 

```r
  q <- function(save="no", ...)  quit(save = save, ...)
```

in my .Rprofile but that does not change the behavior on Ctrl-D

**Comment by klmr** [comment]
Specifying `--no-save` does make a difference:

```
rig run
[…]
> q()
Save workspace image? [y/n/c]: ^D
```

```
R --no-save
[…]
> q()
```

**Comment by AdrienLeGuillou** [comment]
weirdly not in my case
```
I > R --no-save
[..]
> q()
Save workspace image? [y/n/c]: n
```

let me check on another computer

**Comment by AdrienLeGuillou** [comment]
ok apparently that has to do with an alias malfunctioning on my end.
Very sorry for that

**Comment by gaborcsardi** [comment]
Oh, right, so you are running it without any arguments? Then it would indeed make sense to be able to pass some arguments....

**Comment by AdrienLeGuillou** [comment]
Yes, I guess having a way to pass argument into `rig run` to the sub call (R, RScript, etc) could be useful.

I suggested `rig run -- --no-save` as it reminds me of `cargo run a_project -- arguments_to_the_project`

**Comment by klmr** [comment]
Devil’s advocate: make `rig run` an alias for `R --no-save --no-restore` instead of currently just `R`, to shed some historical ballast (the default R settings of saving/restoring environments is daft).

**Comment by gaborcsardi** [comment]
@klmr Yeah, I like that.

**Comment by TymekDev** [comment]
I wouldn't mind having either `--vanilla` or `--no-init-file` and `--no-site-file`. I find it useful when working in a renv project, but wanting do something outside the limited library (e.g. check docs of a package I have installed globally).
